### ISSUE TITLE
getopt trap failing

### ISSUE THREAD

**Issue by calliza** [PapenfussLab/gridss#251]
Hi, 

I could only get version 2.6.2 to run by hashing out the trap commands in gridss.sh. I am on a cluster running CentOS 6.10, Linux version 3.10.0-957.21.3.el6.x86_64 GNU/Linux

Command I used was:
./gridss-2.6.2/scripts/gridss.sh \
        --output test.vcf \
        --assembly test.assembly.bam \
        --reference ./Reference/hs38DH.fasta \
        --threads 2 \
        --jar ./gridss-2.6.2/gridss-2.6.2-gridss-jar-with-dependencies.jar \
        --blacklist ./exclude_intervals.bed \
        --jvmheap 4g \
        --steps All \
        --workingdir ./out \
        --labels U16-B \
        U16-B.final.bam

Regards,
Cali

**Comment by d-cameron** [comment]
I'll need a bit more details to track this down as I don't get any failures on my CentOS 6.4 test machine.

- What was the error message you received?
- What errors/warning were in the full log file (check the output for a line similar to `Mon Sep 30 11:31:54 EST 2019    Running GRIDSS. The full log is in out/gridss.full.20190930_113154.unix305.wehi.edu.au.56718.log`)
- What’s the output from running:
```
env
lsb_release -a
/bin/bash --version
getopt –version
getopt --test
echo $? ${PIPESTATUS[0]}
```

**Comment by calliza** [comment]
Hi Daniel,

Thanks for your fast response.

The error message received was:


getopt --test` failed in this environment.

"exit 1" command completed with exit code 1.


Log file – I do not have this, I attempted to re-generate the error – deleting the hashes I had placed in the grids.sh script - however it is running smoothly today, confusingly.

Command outputs:

$env
#See output attached

$lsb_release -a
LSB Version:    :base-4.0-amd64:base-4.0-noarch:core-4.0-amd64:core-4.0-noarch:graphics-4.0-amd64:graphics-4.0-noarch:printing-4.0-amd64:printing-4.0-noarch
Distributor ID: CentOS
Description:    CentOS release 6.10 (Final)
Release:        6.10
Codename:       Final

$ /bin/bash --version
GNU bash, version 4.1.2(2)-release (x86_64-redhat-linux-gnu)
Copyright (C) 2009 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>

This is free software; you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

$ getopt --version
getopt (enhanced) 1.1.4

$ getopt –test
#No output

$ echo $? ${PIPESTATUS[0]}
4 4

Kind regards,
Cali

Dr Cali Willet
Senior Research Bioinformatics Technical Officer | Sydney Informatics Hub<https://informatics.sydney.edu.au/>
Core Research Facilities<http://sydney.edu.au/research/facilities.html>

Please note I am currently working Wednesdays and Fridays only

THE UNIVERSITY OF SYDNEY
J17 32 Queen Street Chippendale | The University of Sydney | NSW | 2006
P: +61 2 8627 6286
E: cali.willet@sydney.edu.au<mailto:cali.willet@sydney.edu.au>

**Comment by d-cameron** [comment]
Should be fixed in next release (2.6.3)
